docs/user/user/integrating-glean-for-product-managers.md

```diff
@@ -0,0 +1,45 @@
+However, it is also a good idea to have a sense of any additional product-specific questions you are trying to answer with telemetry, and, when possible, in collaboration with a data scientist.  
+This of course helps for your own planning, but is also invaluable for the Glean SDK team to support you, since we will understand the ultimate goals of your product's telemetry and ensure the design will meet those goals and we can identify any new features that may be required.
+It is best to frame this document in the form of questions and use cases rather than as specific data points and schemas.
+
+## Integrating the Glean SDK into your product
+
+The [technical steps for integrating the Glean SDK in your product are documented elsewhere](adding-glean-to-your-project.html) for supported platforms.
+We recommend having a member of the Glean SDK team review this integration to catch any potential pitfalls.
+
+## (Optional) Adapting Glean to your platform
+
+The Glean SDK is a collection of cross platform libraries and tools that facilitate collection of Glean conforming telemetry from applications.  
+Consult the list of the currently supported [platforms and languages](../index.html).
+If your product's tech stack isn't currently supported, please reach out to the Glean team: significant work will be required to create a new integration.  
+In previous efforts, this has ranged from 1 to 3 months FTE of work, so it is important to plan for this work well in advance.
+While the first phase of this work generally requires the specialized expertise of the Glean SDK team, the second half can benefit from outside developers to move faster.
+
+## (Optional) Designing ping submission
+
+The Glean SDK periodically sends telemetry to our servers in a bundle known as a "[ping](../appendix/glossary.html#ping)".  
+For mobile applications with common interaction models, such as web browsers, the Glean SDK provides [basic pings out-of-the-box](pings/index.html).
+For other kinds of products, it may be necessary to carefully design what triggers the submission of a ping.
+It is important to have a solid telemetry plan (see above) so we can make sure the ping submission will be able to answer the telemetry questions required of the product.
+
+## (Optional) New metric types
+
+The Glean SDK has a number of different metric types that it can collect.  
+Metric types provide "guardrails" to make sure that telemetry is being collected correctly, and to present the data at analysis time more automatically.
+Occasionally, products need to collect data that doesn't fit neatly into one of the available metric types.
+Glean [has a process](https://wiki.mozilla.org/Glean/Adding_or_changing_Glean_metric_types) to request and introduce more metric types and we will work with you to design something appropriate.
+This work is generally around 2 weeks FTE work from the Glean team, though we are working on the foundation to accelerate that.
```

Comment:
As an engineer responsible for this of course I'd like to see this promise lowered to "at least" :D 
(but as long as it's clear that this is a rough estimate I'm ok with it)

Reply:
Actually .... should this say something more like "This implementation work ..."?
Proposal & discussion will require 1+ week already.

Reply:
Yeah -- I agree it makes sense to use "at least" here and add some time to include both design + implementation.